src/lib/components/End/After.js

```diff
@@ -6,14 +6,20 @@ const EndAfter = ({
   after,
   handleChange,
 }) => (
-  <div>
-    <input
-      name="end.after"
-      className="form-control"
-      value={after}
-      onChange={numericalFieldHandler(handleChange)}
-    />
-    executions.
+  <div className="col-sm-4">
+    <div className="form-group m-0 row d-flex align-items-sm-center">
```

Comment:
Isn't `.d-flex` excluding usage of `.row` and `.col`?
You use one or the second method of spacing

Reply:
I used flexbox only for aligning items to center (vertically). Basic on what I've seen it doesn't interfere here but there may be easier way for aliging elements center vertically.